waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -42,115 +39,29 @@ migrateDev projectDir maybeMigrationName = do
   --   we are using `script` to trick Prisma into thinking it is running in TTY (interactively).
 
   -- NOTE(martin): For this to work on Mac, filepath in the list below must be as it is now - not wrapped in any quotes.
-  let npxPrismaMigrateCmd = npxPrismaCmd ++ ["migrate", "dev", "--schema", SP.toFilePath schemaFile] ++ optionalMigrationArgs
+  let npxPrismaMigrateCmd = absPrismaExecutableFp projectDir : ["migrate", "dev", "--schema", SP.toFilePath schemaFile] ++ optionalMigrationArgs
```

Comment:
We should change the variable's name to something like `prismaMigrateCmd` (since it no longer uses npx). The same goes for other `npxSomething` variable.

Reply:
Good catch, will update, thanks!